package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "consus-client",
   "description": "The client application for Consus, an inventory management system",
-  "version": "0.1.0",
+  "version": "0.2.0",
```

Comment:
This is coming from changes in dev that made it into this task branch, but not the story branch.

I've merged dev into the story, and this diff should go away after I resolve a merge conflict here...